draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/_styles.scss

```diff
@@ -17,6 +17,78 @@ $ca-navigation-bar__link-item-indicator-height: 5px;
 $ca-navigation-bar__link-item-margin: 18px;
 $ca-navigation-bar__link-item-margin-mobile: 10px;
 
+$ca-breakpoint-desktop-small-up: 1280px;
+$ca-breakpoint-desktop-large-up: 1440px;
```

Comment:
variables in this version of scss are essentially global, so there's a chance this might override or be overridden if the same name is added to the generic `responsive.scss` file. I'd suggest artificially namespacing it with `$navbar-breakpoint-desktop-small-up` or something similar, and to indicate that this is really meant to be private to this file (you really wouldn't want another component reaching in and being dependent on this var, but this version of scss has no way to stop that).

We're hoping to introduce the latest version of scss soon, which actually has a module/namespacing feature, so you wouldn't need to worry about clashes or super long names!

Reply:
Noted!